casepropods/family_connect_registration/tests.py

```diff
@@ -147,3 +226,82 @@ def test_no_http_request_if_contact_uuid_is_none(self):
 
         self.assertEqual(len(responses.calls), 0)
         self.assertEqual(result, {'items': []})
+
+    @responses.activate
+    def test_top_level_results_precendence_over_data(self):
+        responses.add(
+            responses.GET, self.url,
+            json={'results': [{
+                'mama_name': 'sue-toplevel',
+                'data': {
+                    'mama_name': 'sue-data',
+                },
+            }]},
+            match_querystring=True, content_type='application/json')
+
+        responses.add(
+            responses.GET, self.identity_store_url,
+            json={},
+            match_querystring=True, content_type="application/json")
+
+        result = self.pod.read_data({'case_id': self.case.id})
+
+        self.assertEqual(len(result['items']), 13)
+        self.assertEqual(
+            result['items'][0],
+            {'name': 'Mother Name', 'value': 'sue-toplevel'},
+        )
+
+    @responses.activate
+    def test_hub_data_precendence_over_identity_store(self):
```

Comment:
Thinking about it I don't think the hub should have precedence here. A user can change their details outside of a registration so registrations may not be up today. Whereas a new registration should always update the info in the identity store. So the identity store should be considered the source of truth for a user.